docs/installation/_index.md

```diff
@@ -55,37 +61,18 @@ If you are using a microcontroller, prepare your board using the following guide
 
 Other SBCs such as the [RockPi S](https://wiki.radxa.com/RockpiS) and [Orange Pi Zero 2](https://orangepi.com/index.php?route=product/product&path=237&product_id=849) can run Viam with an experimental [periph.io](https://periph.io/) based [modular component](https://github.com/viam-labs/periph_board).
 
-### Install `viam-server`
-
-Once you have a compatible operating system on your board, follow along with the video below or walk through the steps outlined beneath it to install `viam-server` on your board:
-
-{{<youtube embed_url="https://www.youtube-nocookie.com/embed/gmIW9JoWStA">}}
-
-#### Installation steps
+## Install `viam-server`
 
-1. Go to the [Viam app](https://app.viam.com) and [add a new robot](/manage/fleet/robots/#add-a-new-robot).
-   If this is your first time using the Viam app, you must create an account first.
-
-1. On the **Setup** tab, select `Linux (Aarch64)` or `Linux (x86_64)`for the appropriate **Architecture** for your board.
-   On most Linux operating systems, you can run `uname -m` to confirm your board's architecture.
-
-1. Follow the steps shown on the **Setup** tab to install `viam-server` on your board.
-
-1. Once `viam-server` is installed and running on your board, return to the **Setup** page on the [Viam app](https://app.viam.com) and wait for confirmation that your robot has successfully connected.
+Once you have a compatible operating system, follow along with the steps outlined below or with the video beneath it to install `viam-server`:
```

Comment:
Seems like the connection between setup and a compatible OS is not made explicitly clear. Why not keep the info about checking what OS you have in here and maybe include some words about OS in the step above?

Reply:
The info about checking the system is still present in the include - I'll amend this slightly